src/Umbraco.Web/Editors/RelationController.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using Umbraco.Core;
+using Umbraco.Core.Models;
+using Umbraco.Web.Mvc;
+using Umbraco.Web.WebApi.Filters;
+using Constants = Umbraco.Core.Constants;
+
+namespace Umbraco.Web.Editors
+{
+	[PluginController("UmbracoApi")]
+	[UmbracoApplicationAuthorizeAttribute(Constants.Applications.Content)]
+	public class RelationController : ContentControllerBase
```

Comment:
oops! I'm not sure why I inherited from `ContentControllerBase` either.  I did write that bit of code at CG14 Retreat, hmmm ;-)